projects/plugins/jetpack/modules/widgets/rsslinks-widget.php

```diff
@@ -169,32 +200,35 @@ function form( $instance ) {
 			'pink'   => __( 'Pink', 'jetpack' ),
 			'silver' => __( 'Silver', 'jetpack' ),
 		);
-		echo '<p><label for="' . $this->get_field_id( 'imagecolor' ) . '">' . esc_html__( 'Image Color:', 'jetpack' ) . '
-		<select class="widefat" id="' . $this->get_field_id( 'imagecolor' ) . '" name="' . $this->get_field_name( 'imagecolor' ) . '">';
+		echo '<p><label for="' . esc_attr( $this->get_field_id( 'imagecolor' ) ) . '">' . esc_html__( 'Image Color:', 'jetpack' ) . '
+		<select class="widefat" id="' . esc_attr( $this->get_field_id( 'imagecolor' ) ) . '" name="' . esc_attr( $this->get_field_name( 'imagecolor' ) ) . '">';
 		foreach ( $colors as $color => $label ) {
 			echo '<option value="' . esc_attr( $color ) . '"';
-			if ( $color == $image_color ) {
+			if ( $color === $image_color ) {
 				echo ' selected="selected"';
 			}
 			echo '>' . esc_html( $label ) . '</option>' . "\n";
 		}
 		echo '</select></label></p></div>';
 	}
 
-	function _rss_link( $type, $args ) {
-		if ( 'posts' == $type ) {
-			$type_text = __( 'Posts', 'jetpack' );
-			$rss_type  = 'rss2_url';
-		} elseif ( 'comments' == $type ) {
-			$type_text = __( 'Comments', 'jetpack' );
-			$rss_type  = 'comments_rss2_url';
+	/**
+	 * Output a link with a link to the feed.
+	 *
+	 * @param string $type Widget type (posts or comments).
+	 * @param array  $args Widget arguments.
+	 */
+	private function rss_link( $type, $args ) {
```

Comment:
I refactored that method entirely, to make it easier to translate and somehow easier to parse. The current version was very hard to translate as we had translations split in different strings.